coalahtml/_coalahtml/app/controllers/main.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+angular.module('coalaHtmlApp')
+  .controller('MainCtrl', function ($scope, $http) {
+      console.log("In main");
+      var parseCoalaProject = function(projectDir, coalaJSON) {
+          $http.get(projectDir + '/file_data.json')
+            .success(function(file_data) {
+                console.log(file_data);
+                $scope.file_dict = {};
+                Object.keys(file_data).forEach(function(key) {
+                    Object.keys(file_data[key]).forEach(function(file_name) {
+                        if (!(file_name in $scope.file_dict)){
+                          $scope.file_dict[file_name]=file_data[key][file_name];
+                        }
+                    });
+                });
+              coalaJSON = coalaJSON || projectDir + '/coala.json';
+              $scope.data = [];
+              $http.get(coalaJSON).success(function(response){
+                  var knownFiles = {};
```

Comment:
I think we need stricter rules or a codestyle doc (As I'm getting confused by the indentations, some places it's 2 others, it's 4)

Also, some places have space before `{` others done.


Note: Not in this PR ....